Makefile

```diff
@@ -0,0 +1,70 @@
+#!make
+
+# Include config variables for custom course
+include envfile
+export $(shell sed 's/=.*//' envfile)
```

Comment:
I don't understand the intention with `envfile`. Are users supposed to edit it? If not, then it seems to not do very much. If they are supposed to edit it then it looks like it's going to interfere with git?

Reply:
The idea behind the `envfile` was indeed to have users edit it with a location to the user's course. Thus, we avoid having users type out:

```bash
docker run -it --rm -p 3000:3000 -v ~/Google\ Drive/Courses/pl/pl-stat385:/course prairielearn/prairielearn
```

Perhaps, this can be folded into a "template" directory and then on call to a CLI client the `envfile` can be added to `~/` c.f. #1121

Reply:
I'm completely on board with this idea, but I don't like files that are tracked by git but which are supposed to be edited. Can we just ship this without `envfile` committed to git, but still have the Makefile import it if it exists (so a conditional import)?

Reply:
Instead of merging this PR, let's opt to kill it and move over to the `npm` approach mentioned below. This served as a great starting place in terms of features needed.

Reply:
I'm on board with that. I think that we might be able to get away with using npm config variables (see "NPM Config Variables" at https://www.keithcirkel.co.uk/how-to-use-npm-as-a-build-tool/) so users could run:

```
npm set prairielearn:course_dir "/Users/mwest/git/pl-tam212" # once only

npm run pl    # uses the config variable from able
```

(or something like that)